src/sentry/web/frontend/idp_email_verification.py

```diff
@@ -1,18 +1,26 @@
-from django.http.response import HttpResponseNotFound, HttpResponseRedirect
+from typing import Any, Mapping
+
+from django.http.response import HttpResponse
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.auth.idpmigration import get_redis_key, verify_account
 from sentry.utils.auth import get_login_url
+from sentry.web.helpers import render_to_response
+
+
+def _respond(
+    template: str,
+    context: Mapping[str, Any] = None,
+    status: int = 302,
+) -> HttpResponse:
+    return render_to_response(template, context, status=status)
 
 
 def idp_confirm_email(request: Request, key: str) -> Response:
     verification_key = get_redis_key(key)
```

Comment:
i think it would be a little cleaner if we simply set the `key` on the django session and formatted it when we want to read with it -- if ever switched off of redis for example, this could complicate that.